docs/references/nextjs/route-protection.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Route protection | Next.js
+description: Learn how to protect the routes in your application 
+---
+
+# Route protection
+
+Clerk offers you, the developer, any options to secure the routes in their applications, as well as easy ways to integrate role or permissions based checks. The opt-in nature of Clerk gives you flexibility in how you want to build you application.
+
+
+## Core concepts
+
+### Opt-In 
+
+`clerkMiddleware()` provides opt-in protection for your routes. By default no routes will be protected. You will indicate what routes to protect through the Middleware, layouts or even per route/page. 
+
+### Intregration with Organizations and Roles
+
+`clerkMiddleware()` provites a new `auth().protect()` method that can be used when creating rules to protect pages. This will integrate with Organization role and permissions, allowing you to protect routes based on the permissions or roles a user has.
+
+### Protect via Middleware, routes, pages, or a mixture of all three
+
+With the opt-in approach of `clerkMiddleware()` you are free to protect your routes and pages in the most conveniant way to you.
+
+
+## Middleware only
+
+Does your application have limited or straight forward route protection needs? Configuring your protection in Middleware and only Middleware could be the play. Perhap your application is only available to signed in users, or most of your app is under the `/dashboard` route and only available for signed in users. In this case protecting from Middleware is a central location that enables this protection.
```

Comment:
"could be the play" is a colloquiums that might not translate well internationally in different languages. 
```suggestion
Does your application have limited or straightforward route protection needs? Configuring your protection in Middleware and only Middleware could be the right choice for you. Perhaps your application is only available to signed in users, or most of your app is under the `/dashboard` route and only available for signed in users. In this case protecting from Middleware is a central location that enables this protection.
```